gemelli/scripts/__init__.py

```diff
@@ -1,3 +1,22 @@
-from ._standalone_ctf import standalone_ctf
+import click
+from gemelli import __version__
+from importlib import import_module
 
-__all__ = ["standalone_ctf"]
+
+def _terribly_handle_brokenpipeerror():
+    # based off http://stackoverflow.com/a/34299346
+    import os
+    import sys
+    sys.stdout = os.fdopen(1, 'w')
+
+
+@click.group()
+@click.version_option(version=__version__)
+@click.pass_context
+def cli(ctx):
+    ctx.call_on_close(_terribly_handle_brokenpipeerror)
+
+
+import_module('gemelli.scripts._standalone_transforms')
```

Comment:
I am not sure I understand these imports

Reply:
[This link](https://stackoverflow.com/questions/34643620/how-can-i-split-my-click-commands-each-with-a-set-of-sub-commands-into-multipl) may be useful for avoiding the circular import anti-pattern